### DISCUSSION TITLE
Error on registering Microsoft Azure Student using GitHub

### DISCUSSION THREAD

**Discussion by AbhinavTheDev** [community/community#147858] [GitHub Education]
I am already verified in GitHub student developer pack, when I try to register on Microsoft Azure through my GitHub account so an error is occurring. Can anyone please help me to resolve this issue.

![20241224_144304.jpg](https://github.com/user-attachments/assets/c9850d5f-0db8-4ff3-80c2-791ae5bcea8c)

![Screenshot_20241224_154731.jpg](https://github.com/user-attachments/assets/89cb43e1-5d43-4c05-a422-022188b37868)

![IMG_20241224_155237.jpg](https://github.com/user-attachments/assets/21c2091d-14d8-46f1-b760-fc3cd795f64c)

**Comment by davevad93** [comment]
Hi @AbhinavTheDev ,

This is a known issue that has been solved here #135739 , you have the following options:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I recommend the first one.

**Reply by AbhinavTheDev** [reply]
I already opened a ticket in GitHub support [#3154390](https://support.github.com/ticket/personal/0/3154390)

But I still don't get any solution yet.

**Reply by davevad93** [reply]
Hey @AbhinavTheDev , support can take a while to respond because they have to deal with many daily requests, so it's normal that you haven't received any response yet. Be patient because they can take days and in some cases, even weeks! 👍
